docs/src/modules/components/AppNavDrawer.js

```diff
@@ -172,22 +177,83 @@ function AppNavDrawer(props) {
+                          ? theme.palette.primaryDark[600]
+                          : theme.palette.grey[200]
+                      }`,
+                    color: (theme) =>
+                      theme.palette.mode === 'dark'
+                        ? theme.palette.primary[300]
+                        : theme.palette.primary[500],
+                    mr: 2,
+                  }}
+                >
+                  {/* eslint-disable-next-line material-ui/no-hardcoded-labels -- version string is untranslatable */}
+                  {`v${process.env.LIB_VERSION}`}
+                </Button>
+                <Menu
+                  anchorEl={anchorEl}
+                  open={Boolean(anchorEl)}
+                  onClose={() => setAnchorEl(null)}
+                  PaperProps={{
+                    variant: 'outlined',
+                    sx: {
+                      mt: 0.5,
+                      minWidth: 160,
+                      borderColor: (theme) =>
+                        theme.palette.mode === 'dark' ? 'primaryDark.700' : 'grey.200',
+                      bgcolor: (theme) =>
+                        theme.palette.mode === 'dark' ? 'primaryDark.900' : 'background.paper',
+                      boxShadow: (theme) =>
+                        `0px 4px 20px ${
+                          theme.palette.mode === 'dark'
+                            ? alpha(theme.palette.background.paper, 0.72)
+                            : 'rgba(170, 180, 190, 0.3)'
+                        }`,
+                      '& .MuiMenuItem-root': {
+                        fontSize: (theme) => theme.typography.pxToRem(14),
+                        fontWeight: 500,
+                        '&.Mui-selected': {
+                          color: 'primary.main',
+                        },
+                      },
+                    },
+                  }}
+                >
+                  <MenuItem selected onClick={() => setAnchorEl(null)}>
+                    {/* eslint-disable-next-line material-ui/no-hardcoded-labels -- version string is untranslatable */}
+                    {`v${process.env.LIB_VERSION}`} <DoneRounded sx={{ fontSize: 16, ml: 0.25 }} />
+                  </MenuItem>
+                  <MenuItem
+                    component="a"
+                    href={`https://v4.mui.com${languagePrefix}/`}
+                    onClick={onClose}
+                  >
+                    {/* eslint-disable-next-line material-ui/no-hardcoded-labels -- version string is untranslatable */}
+                    {`v4`}
+                  </MenuItem>
+                  <Divider />
+                  <MenuItem
+                    component="a"
+                    href={`https://mui.com${languagePrefix}/versions/`}
+                    onClick={onClose}
+                  >
+                    {/* eslint-disable-next-line material-ui/no-hardcoded-labels -- version string is untranslatable */}
```

Comment:
We could translate it. I guess we can wait for either we fix the sync with Crowdin, resuming the translations, or we give up on translating the docs.

Reply:
I can translate it to Portuguese 😬 Guess you could do it in French?!